### PR TITLE
Do not show the "Sign up for free" button in issue tracker

### LINKED ISSUE #3407: "Sign Up For Free" link visible in issues even if registration is disabled
When visiting an issue, even if registration is disabled, there's a "Sign up for free" green button.
The culprit is templates/repo/issue/view_content.tmpl having an unconditioned inclusion of the repo.issues.sign_up_for_free text and block.

I've tried fixing it but the localized strings are intermixed between "repo.issues.sign_up_for_free" (having the first portion of the sentence) and "repo.issues.sign_in_required_desc" having the second portion.

Cannot reproduce on try.gogs.io because there the registration is enabled.
Current master==develop==90af997fecb36e5cd1863f5d724a9df7a0a04118 are affected (0.9.68.0807)

### PR DESCRIPTION
**Pull request by strk**
The "Sign in to comment" link is good enough and will correctly
show or not show the "Sign Up" button link for those not having
an account already.

Fixes #3407 (link to nowhere when registration is disabled)

### CODE REVIEW

File: conf/locale/locale_en-US.ini [hunk 1]
```diff
@@ -485,8 +485,7 @@ issues.commit_ref_at = `referenced this issue from a commit <a id="%[1]s" href="
 issues.poster = Poster
 issues.collaborator = Collaborator
 issues.owner = Owner
-issues.sign_up_for_free = Sign up for free
-issues.sign_in_require_desc = to join this conversation. Already have an account? <a href="%s">Sign in to comment</a>
+issues.sign_in_require_desc = <a href="%s">Sign in to comment</a>
```
Comment (Unknwon) [inline conf/locale/locale_en-US.ini]: In this case, I think we should put `<a></a>` part directly in the template, and only keep translate words in locale file.

Comment (strk) [inline conf/locale/locale_en-US.ini]: Agreed, but as usual, I try to keep my PR focused on a topic
(hiding the "Sign up for free" part, in this case).

I'm leaving for holiday so won't be able to improve this PR
any soon.

Comment (Unknwon) [inline conf/locale/locale_en-US.ini]: You can fix it when you come back.

**Comment by strk** [comment]
I've pushed a new commit to remove markup from the translated strings, moving them to the template. And I rebased to develop branch.

File: templates/repo/issue/view_content.tmpl [hunk 2]
```diff
@@ -227,8 +227,7 @@
 				</div>
 			{{else}}
 				<div class="ui warning message">
-					<a href="{{AppSubUrl}}/user/sign_up" class="ui green button">{{.i18n.Tr "repo.issues.sign_up_for_free"}}</a>
-					{{.i18n.Tr "repo.issues.sign_in_require_desc" .SignInLink | Safe}}
+					<a href="{{ .SignInLink }}">{{.i18n.Tr "repo.issues.sign_in_require_desc" | Safe}}</a>
```
Comment (Unknwon) [inline templates/repo/issue/view_content.tmpl]: Remove extra space in `{{ .SignInLink }}`.

**Pull request merged**